packages/starrating/src/react/__specs__/index.spec.js

```diff
@@ -1,80 +1,57 @@
+import { render } from '@testing-library/react'
 import React from 'react'
-import { mount } from 'enzyme'
 
-import StarRating from '..'
+import StarRating from '../index.js'
```

Comment:
Apparently we've all been missing style updates in quite a few spec files...

Reply:
yeah. i think we're pretty close to the fated "update all the lint errors" commit. no rush though